python/doc/theory/reliability_sensitivity/taylor_moments.rst

```diff
@@ -1,144 +1,106 @@
-
-| **Case** :math:`n_Y=1`
+.. _taylor_expansion_moments:
 
-.. math::
+Taylor expansion moments
+------------------------
 
-    Y = h(\muX) + < \underline{\nabla} h(\muX) , \: \uX - \muX> + \frac{1}{2}<<\underline{\underline{\nabla }}^2 h(\muX,\: \underline{\mu}_{\:X}),\: \uX - \muX>,\: \uX - \muX> + o(\Cov \uX)
+One way to evaluate the central dispersion (expectation and variance) of the variable :math:`\uY=h(\uX)`
+is to use the Taylor expansion of the function :math:`h: \Rset^d \rightarrow \Rset^q` at
+the mean point :math:`\vect{\mu} = \Expect{\uX}`.
 
-where:
+Depending on the order of the Taylor expansion (classically first or second order), we get different approximations
+of the mean and variance of :math:`Y`.
 
--  :math:`\muX = \Expect{\uX}` is the vector of the input variables at
-   the mean values of each component.
 
--  :math:`\Cov \uX` is the covariance matrix of the random vector
-   :math:`\uX`. The elements are the followings :
-   :math:`(\Cov \uX)_{ij} = \Expect{\left(X^i - \Expect{X^i} \right)\times\left(X^j - \Expect{X^j} \right)}`
+In the remainder, we denote by :math:`\Cov \uX`  the covariance matrix of :math:`\uX`, defined by:
 
--  :math:`\underline{\nabla} h(\muX) = \: ^t \left( \frac{\partial y}{\partial x^j}\right)_{\ux\: =\: \muX} = \: ^t \left( \frac{\partial h(\ux)}{\partial x^j}\right)_{\ux\: =\: \muX}`
-   is the gradient vector taken at the value :math:`\ux = \muX` and
-   :math:`j=1,\ldots,n_X`.
+.. math::
 
--  :math:`\underline{\underline{\nabla}}^2 h(\ux,\ux)` is a matrix. It
-   is composed by the second order derivative of the output variable
-   towards the :math:`i^\textrm{th}` and :math:`j^\textrm{th}`
-   components of :math:`\ux` taken around :math:`\ux = \muX`. It yields
-   to:
-   :math:`\left( \nabla^2 h(\muX,\muX) \right)_{ij} =\left( \frac{\partial^2 h(\ux,\ux)}{\partial x^i \partial x^j}\right)_{\ux\: =\: \muX}`
+    \Cov \uX = (c_{ij})_{ij}
 
--  :math:`<,>` is a scalar product between two vectors.
 
-| **Approximation at the order 1 - Case** :math:`n_Y=1`
+where:
 
-  .. math::
+.. math::
 
-      \Expect{Y} = h(\muX)
+    c_{ij} = \Expect{\left(X_i - \Expect{X_i}\right)\left(X_j - \Expect{X_j} \right)}
 
 
-  .. math::
+with :math:`c_{ii} = \sigma_i^2`.
 
-      \Var{Y} = \sum_{i,j=1}^{n_X} \frac{\partial h(\muX)}{\partial X^i} . \frac{\partial h(\muX)}{\partial X^j}.(\Cov \uX)_{ij}
+**Case 1:** :math:`Y \in \Rset` , :math:`q=1`:
 
-| **Approximation at the order 2 - Case** :math:`n_Y=1`
+The second order Taylor expansion of :math:`h` around :math:`\ux = \vect{\mu}` is written as:
 
-  .. math::
+.. math::
 
-     \begin{aligned}
-         \begin{split}
-           \Expect{Y} = h(\muX) + \frac{1}{2}. \sum_{i,j=1}^{n_X} \frac{\partial^2 h(\muX,\muX)}{\partial x^i \partial x^j} . (\Cov \uX)_{ij}
-         \end{split}
-       \end{aligned}
+    Y = h(\vect{\mu}) + \sum_{i = 1}^d \left( \frac{\partial h}{\partial x_i }\right)(\vect{\mu})(X_i-\mu_i)
```

Comment:
```suggestion
    h(\vect{X}) = h(\vect{\mu}) + \sum_{i = 1}^d \left( \frac{\partial h}{\partial x_i }\right)(\vect{\mu})(X_i-\mu_i)
```

Reply:
Yes you are right. I changed Y into y and X into x.

Reply:
As far as I can read, you replaced Y with the lower case letter y. This is not correct: the left hand side is really h(x) in the Taylor expansion. I fixed that in #2450.